platform/security/security-review-process.md

```diff
@@ -0,0 +1,25 @@
+# Security review process
+
+## Context
+
+Before launching an application on VA.gov, the feature should be reviewed by a VSP security engineer.
+
+This isn't intended to be a thorough audit, but a high-level overview of your application, with a focus on any sensitive areas.
+
+**Note: this is extremely low-process today and will be more fleshed out in the future.**
+
+## Process
+
+1. File a [GitHub issue in `va.gov-team`](https://github.com/department-of-veterans-affairs/va.gov-team/issues/new):
+    * Provide the title "Security Review for [feature name]"
+    * Tag with the label `security-review`
+    * Provide the following in the ticket:
+        * an architecture diagram, showing involved systems + data flows
+        * description of any new publicly-exposed endpoints (`vets-api` or otherwise)
+        * description of any new interactions with dependent services (i.e. `vets-api` --> ???)
+        * description of any other security hotspots you're concerned about / want extra attention on
```

Comment:
Should we put the concerns in `va.gov-team-sensitive`?  We probably don't want to announce our concerns of vulnerabilities.

Reply:
ooh good question! I'd be supportive of that change.

@andreahewitt-odd do you think it's OK process-wise to move security review issues into `va.gov-team-sensitive`?